src/components/Global/Header/Header.module.scss

```diff
@@ -0,0 +1,76 @@
+$white: white;
+$black: black;
+$primary-color: #F16B67;
+
+.header {
+    width: 100vw;
+    height: 7vh;
+    background-color: $white;
+    padding: 0 2vw;
+
+    .content {
+        width: 100%;
+        height: 100%;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        align-items: center;
+    
+        .left {
+            display: flex;
+            flex-wrap: wrap;
+            align-items: center;
+
+            p {
+                font-size: 1.4rem;
+                font-weight: 700;
+                margin-left: 1rem;
+            }
+
+            .tabs {
+                margin-left: 5rem;
+                cursor: pointer;
+                display: flex;
+                align-items: center;
+
+                span {
+                    height: 6vh;
```

Comment:
same thing, this should probably be based off of the height of the header instead of a set value, since it's a component inside of the header